lib/ShapesGraph.ts

```diff
@@ -367,6 +367,16 @@ export class ShapesGraph {
     nodeShapeId: Term,
     shape: ShapeTemplate,
   ) {
+    // Extract label
+    const rdfsLabel = getObjects(shapeStore, nodeShapeId, RDFS.terms.label)[0];
```

Comment:
@julianrojas87 Doesn't this give an error when no objects is found?

Reply:
Apparently it doesn’t

Reply:
`getObjects()` returns an empty array if no match is found. So I think the code is safe as it is.